codespell_lib/data/dictionary.txt

```diff
@@ -991,7 +1003,11 @@ begginings->beginnings
 beggins->begins
 begining->beginning
 beginnig->beginning
-behaivior->behaviour
+behaivior->behavior, behaviour,
+behaviour->behavior, behaviour,
```

Comment:
`behaviour`is british spelling. Do we want to have this? Or should we set up a flag to include british spelled words ?

Reply:
I'd rather leave handling of British spellings to a different discussion/PR than one that aims to simply add a lot of words

Reply:
The dictionary that I'm converting orients to british spelling. So how do you wish I proceed. Should we just add everything and then sort it out after the migration ?

Reply:
> I'd rather leave handling of British spellings to a different discussion/PR than one that aims to simply add a lot of words

Makes sense. I'm just not sure how to proceed. Should I: 
1) take out all the British spelling typos? Or  
2) take out both British typos and solutions?  
Ex.  
1) Remove just `behaviour->behavior, behaviour,` Or  
2) Commit the following instead:    
behaivior->behavior, ~behaviour~,
~behaviour->behavior, behaviour~,
behavoir->behavior, ~behaviour~,
behavour->behavior, ~behaviour~,
behavoirs->behaviors, ~behaviours~,

Reply:
I'll just remove British words altogether and add them to #103

Reply:
@larsoner ping